Core/Core/Files/FilePicker/FilePickerViewController.swift

```diff
@@ -362,3 +363,16 @@ extension FilePickerViewController: VNDocumentCameraViewControllerDelegate {
         }
     }
 }
+
+extension FilePickerViewController: FilePickerCellDelegate {
+    func removeFile(_ file: File) {
+        let title = NSLocalizedString("Remove File", bundle: .core, comment: "")
+        let message = NSLocalizedString("Are you sure you want to remove this file from the submission?", bundle: .core, comment: "")
```

Comment:
I think this File Picker gets used for more than just assignment submissions, so we may want to remove the submission-specific text.

@armstrongnate Would `"Are you sure you want to remove this file?"` be ok?

Reply:
Yes, you're right. That would be better.